ServerCore/Pages/Resources/Default/FAQPartial.cshtml

```diff
@@ -0,0 +1,3 @@
+﻿<div class="container">
+    <h2>Default FAQ!</h2>
```

Comment:
wow....you take the bunnies out of this and then have the audacity to request a review....


...thankfully you have so much goodwill stocked up from making the demo event rabbit-themed that I can let this slide :P